release.json

```diff
@@ -17,9 +17,9 @@
         "JMXFETCH_VERSION": "0.44.0",
         "JMXFETCH_HASH": "758717a2fd9c50247336318644e9c985d9a8a9845039d59a8793923b3f76f494",
         "MACOS_BUILD_VERSION": "master",
-        "WINDOWS_DDNPM_DRIVER": "release-signed",
-        "WINDOWS_DDNPM_VERSION": "0.98.2.git.74.86e7c5b",
-        "WINDOWS_DDNPM_SHASUM": "61e13fccdca7bf6a356e7ef552a7845316b9905b5eee264811982e3034a3bb84",
+        "WINDOWS_DDNPM_DRIVER": "testsigned",
```

Comment:
Is this a desired change for the freeze?

Reply:
that was temporary.  Now have a signed driver.